discovery-provider/plugins/network-monitoring/src/discovery/queries.ts

```diff
@@ -112,7 +115,10 @@ export const importUsers = async (run_id: number) => {
             :run_id,
             primary_id as primarySpID, 
             secondary_ids[1] as secondary1SpID,
-            secondary_ids[2] as secondary2SpID
+            secondary_ids[2] as secondary2SpID,
+            -2,
+            -2,
+            -2
```

Comment:
what do these signify?

Reply:
Before, when importing users from discovery DB, it would default set the clock values to `null`. This isn't inherently bad but it was making it hard to debug and differentiate between users that got skipped and users that legitimately had a null clock value. I would set it to `-1` but I'm already using `-1` for when there's an error fetching a user's clock value so I set it to `-2`.

In short, a clock value of `-2` just means it hasn't had its clock value fetched from content nodes yet.

Reply:
interesting.......... ok! sure. can we add a comment here :)

Reply:
Done

Reply:
+1 to vicky's Q - this def feels weird

how could a user have a "null" clock value? clock status route always returns a non-null value. any reason to not import with all values as -1 or null, and when users get skipped, just leave it as is? -1 is returned by clock status when that node has no value for user, so maybe we can import as null?

Reply:
I'll change it